package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
@dlebu Do u know if VS is using the `peerDependencies` in `package.json`? Otherwise I think the right fix is to just get rid of all the `peerDependencies` since `plugin.xml` should be sufficient for Cordova.